src/main/java/com/zendesk/maxwell/Maxwell.java

```diff
@@ -73,7 +52,8 @@ private void run(String[] argv) throws Exception {
 		AbstractProducer producer = this.context.getProducer();
 		AbstractBootstrapper bootstrapper = this.context.getBootstrapper();
 
-		final MaxwellReplicator p = new MaxwellReplicator(this.savedSchema, producer, bootstrapper, this.context, this.context.getInitialPosition());
+		MysqlSchemaStore mysqlSchemaStore = new MysqlSchemaStore(this.context);
```

Comment:
I think it's the same.  The only reason that we catch & return `SQLException` (instead of just letting `main` log it before exiting) is that it's very common to misconfigure your SQL connection, so we generally don't throw a stack trace in there.